src/styles/_variables.scss

```diff
@@ -17,26 +20,24 @@ $color-button-hovered: #005d92;
 // Light:
 $color-text-light: rgba(black, 0.85);
 $color-text-light-alt: rgba(white, 0.85);
-$color-bg-light-1: #ccd1d6;
-$color-bg-light-2: #e6e8eb;
+$color-accent-purple: #a000bc;
+$color-bg-light-1: #ffffff;
+$color-bg-light-2: #f8f9fa;
 $color-bg-light-3: #f0f1f3;
-$color-bg-light-4: #f8f9fa;
-$color-bg-light-5: #eeeeee;
+$color-bg-light-4: #e6e8eb;
+$color-bg-light-5: #dadee2;
```

Comment:
I would recommend adding a note to state that the naming of these colors are adopted and should follow that from the styleguide to prevent them to be updated or overwritten by accident without updating the styleguide in the future.

Reply:
and also to add a comment on line 5 to remind whoever that might come across this file the need to check and align with the styleguide :D